CryptoSlam integration

khrystyna-CS opened the pull request:
# Grant Application Checklist

- [x] The [`applications/application-template.md`](../applications/application-template.md) or [`applications/research-template.md`](../applications/research-template.md) has been copied, renamed ( "cryptoslam_integration.md") and updated.
- [x] An Ethereum address for the payment of the milestones is provided inside the application.  
- [x] The total funding amount of the project is up to USD $30k for every grant applications.
- [x] The initial PR contains only one commit (squash if needed before submitting your PR).
- [x] If you own an Economic X Node, make sure you have already sent an email([`applications/email-template.md`](../applications/email-template.md)) to grant@vechain.org to claim the additional grant.

NecoSherry commented:
Thanks for your application.We had a quick look and found some problems:
1. The maximum amount of our grant program is $30,000. We recommend you reduce the price.  If you want higher funds, I'd recommend you to directly contact the Foundation to see if there is any opportunity.
2. We will not fund the entire grant in a lump sum  in the first phase. Please divide the funds among the milestones according to the workload.
3. The milestone, duration and price in your application do not match. We have provided an [example roadmap](https://github.com/vechain/grant-program/blob/master/applications/application-template.md#development-roadmap) in the application template. You need to reorgnize the roadmap and provide detailed deliverables referring to the template.

khrystyna-CS commented:
Hi NecoSherry,
I just updated the application.
Please let me know if you have questions or need more details.

Looking forward to hearing back from you!

Commit number: 9cc63b156639040006c72ffc371f69e0333798a0

NecoSherry commented:
Hi @khrystyna-CS, thanks for your update. We list a few improvements that could help to increase the chance of acceptance:
1. Your pull request modified our README.md. Could you please change this and update `cryptoslam_integration.md` you created in the earlier commit?
2. For the milestones, once the application is accepted, the payment for the first milestone will be immediately disbursed to fund the project. We usually try to lower the risk (not the monetary one, but the risk of misunderstandings and disputes) by asking for small milestones, so that feedback and conversations happen along the way. We recommend you restructure the roadmap and split your milestones. 
3. Note that deliverables should be tangible and could be verified. If the deliverables are functions in your platform, we require documentation (tutorial, demo video or user guide with main screenshots) for each deliverable. Whenever milestones are delivered, we refer to this document to ensure that everything has been delivered on your website as expected.
4. Could you please provide more information about your team members? Describe the team's relevant experience and list their LinkedIn if they have.

khrystyna-CS commented:
Hey NecoSherry, 
How are you?
Thanks for getting back to us and for the feedback.

I've edited the README and updated cryptoslam_integration.md
(Points 2 and 3) I discussed the milestones with our engineers before committing to previous changes, and they mentioned that, unfortunately, there would be tangible items once we only complete the onboarding of collections.
Please share why you need our engineers' information. If you agree to work with CryptoSlam, I recommend scheduling a dev call between VeChain and CryptoSlam teams to discuss the details. 

Looking forward to hearing back from you!

Thnx,
Khrystyna 


> On Jan 30, 2023, at 4:34 AM, NecoSherry ***@***.***> wrote:
> 
> 
> Hi @khrystyna-CS <https://github.com/khrystyna-CS>, thanks for your update. We list a few improvements that could help to increase the chance of acceptance:
> 
> Your pull request modified our README.md. Could you please change this and update cryptoslam_integration.md you created in the earlier commit?
> For the milestones, once the application is accepted, the payment for the first milestone will be immediately disbursed to fund the project. We usually try to lower the risk (not the monetary one, but the risk of misunderstandings and disputes) by asking for small milestones, so that feedback and conversations happen along the way. We recommend you restructure the roadmap and split your milestones.
> Note that deliverables should be tangible and could be verified. If the deliverables are functions in your platform, we require documentation (tutorial, demo video or user guide with main screenshots) for each deliverable. Whenever milestones are delivered, we refer to this document to ensure that everything has been delivered on your website as expected.
> Could you please provide more information about your team members? Describe the team's relevant experience and list their LinkedIn if they have.
> —
> Reply to this email directly, view it on GitHub <https://github.com/vechain/grant-program/pull/121#issuecomment-1408267336>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/A43N7SUWUYFBRBI77KWCS7LWU6DLNANCNFSM6AAAAAATEZA2HA>.
> You are receiving this because you were mentioned.
>

NecoSherry commented:
Hi @khrystyna-CS , thank you for your quick response and the update. At this preliminary review stage, we simply wanted to communicate with the applicant and make suggestions to help make the application as comprehensive as possible to improve the chances of it being approved by the committee. We usually hope to obtain team information so that our committee can assess the feasibility of the project. 

I understand that you may not feel the need to provide any additional information beyond what you have already supplied, and that is perfectly fine. If you feel the current content is sufficient, please do let me know so that the application can be moved to the committee review stage. The final decision will be made by the committee. 

However, if you feel that you can provide more information, we would be more than happy to review it. If you would like to provide additional, more confidential information that may not be suitable for publication on GitHub, please feel free to email it to grant@vechain.org.

khrystyna-CS commented:
Hi,

Thank you for your help and recommendations!

I’ve spoke with my team and we want to proceed with the current application as it is. Can you please move it to the committee review stage?
We are looking forward to work with VeChain!

Thank you,
Khrystyna

> On Feb 1, 2023, at 4:18 AM, NecoSherry ***@***.***> wrote:
> 
> 
> Hi @khrystyna-CS <https://github.com/khrystyna-CS> , thank you for your quick response and the update. At this preliminary review stage, we simply wanted to communicate with the applicant and make suggestions to help make the application as comprehensive as possible to improve the chances of it being approved by the committee. We usually hope to obtain team information so that our committee can assess the feasibility of the project.
> 
> I understand that you may not feel the need to provide any additional information beyond what you have already supplied, and that is perfectly fine. If you feel the current content is sufficient, please do let me know so that the application can be moved to the committee review stage. The final decision will be made by the committee.
> 
> However, if you feel that you can provide more information, we would be more than happy to review it. If you would like to provide additional, more confidential information that may not be suitable for publication on GitHub, please feel free to email it to ***@***.*** ***@***.***>.
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/vechain/grant-program/pull/121#issuecomment-1411720119>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/A43N7STSNERC4VQSXSNFJGDWVIS7RANCNFSM6AAAAAATEZA2HA>.
> You are receiving this because you were mentioned.
>

NecoSherry commented:
Hi @khrystyna-CS Congratulations! Your application has been accepted by the committee! 

We have recently updated our payment currency to USDC (ERC 20). Therefore we would kindly ask you to update the payment address in your application form to USDC(ERC 20) accordingly. We will perform a small test payment before the first payment.

In order to proceed with further payments, we need to contact you by email. Please send an email to grant@vechain.org with your project name, PR link and new payment address (same as in the PR). 

We look forward to hearing from you.

Lalsha40 commented:
I need pr no
